src/cli.ts

```diff
@@ -125,7 +132,7 @@ async function main({
   const genPath = output || mktemp();
   try {
     const generator = new TSOutput(genPath);
-    const generated = await generate(runtime, paths, { client, server });
+    const generated = await generate(runtime, paths, { client, server }, { noExtraProps: !allowExtraProps });
```

Comment:
Should we be able to generate separate schemas for client and server? Otherwise in our current architecture when we generate both at once we will end up with different packages used for tests and for production? Or it's acceptable to use the strict version in tests and the loose version in production?

Reply:
We agreed that we'll publish clients that allow extra props and servers that don't.
We only generate both at once in the parent package, I suggest we disallow extra props in that case.
https://github.com/binaris/nodeutils/pull/227